ImageData/HDF5Loader.h

```diff
@@ -68,6 +69,11 @@ typename HDF5Loader::image_ref HDF5Loader::loadData(FileInfo::Data ds) {
     return dataSets[data];
 }
 
+bool HDF5Loader::getPixelMaskSlice(casacore::Array<bool>& mask, const casacore::Slicer& slicer) {
+    // HDF5Lattice is not a masked lattice
```

Comment:
Possibly another reason for us to use an `HDF5Image`?

Reply:
Yes, there are many reasons to use HDF5Image and casacore Images in general.  Will do this as soon as I finish the last bit of lattice region implementation.

Reply:
OK. But please hold off until after the hdf5 stats branch has been merged -- it makes a lot of changes to the HDF5 loader code (although this does not affect the current type of the main HDF5 image object).

Reply:
Sorry, saw your comment after merge was done.

Reply:
Don't worry; I meant hold off on making changes to the image type created by the HDF5 loader. This merge is fine.